multiplayer/src/services/gameClient.ts

```diff
@@ -131,27 +127,10 @@ class GameClient {
 
         const authToken = await authClient.authTokenAsync();
 
-        // TODO: Send real credentials
```

Comment:
Do we still need this `send real credentials` to do? Or is that done already?

Reply:
Yes, we're sending actual credentials now.